Microsoft.Toolkit.Uwp.SampleApp/SamplePages/LayoutTransformControl/LayoutTransformControlXaml.bind

```diff
@@ -0,0 +1,29 @@
+<Page
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+  <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <controls:LayoutTransformControl Background="{StaticResource Brush-Grey-01}"
+                                     HorizontalAlignment="Center"
+                                     VerticalAlignment="Center"
+                                     RenderTransformOrigin="0.5,0.5">
+      <controls:LayoutTransformControl.Transform>
+        <TransformGroup>
+          <RotateTransform Angle="@[Angle:DoubleSlider:0:-180.0-180.0]" />
+          <ScaleTransform ScaleX="@[ScaleX:DoubleSlider:1:0.0-5.0]"
+                          ScaleY="@[ScaleY:DoubleSlider:1:0.0-5.0]" />
+          <SkewTransform AngleX="@[SkewX:DoubleSlider:0:-180.0-180.0]"
+                         AngleY="@[SkewY:DoubleSlider:0:-180.0-180.0]" />
+        </TransformGroup>
+      </controls:LayoutTransformControl.Transform>
```

Comment:
Agree with @skendrot, could be useful to show more of what it's doing compared to a RenderTransformOrigin.

I did notice that with similar values in Skew though that at some point the layout transform seems to start behaving differently...

![image](https://user-images.githubusercontent.com/24302614/37811098-3dfc20a0-2e15-11e8-96a6-bb90c6c7f33b.png)

```
    <Grid Background="{StaticResource Brush-Grey-01}" Grid.Row="1"
          HorizontalAlignment="Center" VerticalAlignment="Center"
          RenderTransformOrigin="0.5,0.5">
        <Grid.RenderTransform>
          <TransformGroup>
            <RotateTransform Angle="-2.25" />
            <ScaleTransform ScaleX="2.21"
                          ScaleY="2.2" />
            <SkewTransform AngleX="-120"
                           AngleY="180" />
          </TransformGroup>
        </Grid.RenderTransform>
      <Grid HorizontalAlignment="Center" VerticalAlignment="Center" Margin="10">
        <TextBlock Foreground="White" Text="This is a test message." />
      </Grid>
    </Grid>

```

Compared to when there are less large values:

![image](https://user-images.githubusercontent.com/24302614/37811121-639721fc-2e15-11e8-8771-8bdc93ff7473.png)

Would this be a bug somewhere?

Reply:
Well. Your example is much more complicated than the code you gave. One thing I can notice is that in your large example, the first rendered grid is correct but not the second one.

Reply:
@Odonno both images are the same example, only difference is the change in Skew Angles (-20, 20) for the bottom and (-120, 180) for the top.  The top rectangle in each image is the Layout Transform and the bottom rectangle is the Render Transform.  I just added it into the example as another row of the main grid.

Reply:
I have achieved to replicate the second render and here is the configuration I set:

```
<controls:LayoutTransformControl Background="{StaticResource Brush-Grey-01}"
                                     HorizontalAlignment="Center"
                                     VerticalAlignment="Center"
                                     RenderTransformOrigin="0.5,0.5">
      <controls:LayoutTransformControl.Transform>
        <TransformGroup>
          <RotateTransform Angle="-2.25" />
          <ScaleTransform ScaleX="2.21"
                        ScaleY="2.2" />
          <SkewTransform AngleX="80"
                          AngleY="0" />
        </TransformGroup>
      </controls:LayoutTransformControl.Transform>

        <Grid HorizontalAlignment="Center" VerticalAlignment="Center" Margin="10">
        <TextBlock Foreground="White" Text="This is a test message." />
      </Grid>
    </controls:LayoutTransformControl>
```

Reply:
@michael-hawker Oh, I see. It is not the same method you used. I suppose that's maybe because the Matrix is not altered in the same order as it is with the default RenderTransform.

Reply:
@michael-hawker Ok. Sorry. My mistakes. It seems angles are in the opposite direction. So, `AngleX` of +180 should be -180 on the other `SkewTransform`.

Reply:
@xyzzer may have an idea.

Reply:
Not time to look into it now.
Here's the sample I provided in WRTXT:
https://github.com/xyzzer/WinRTXamlToolkit/blob/master/WinRTXamlToolkit.Sample/Views/Controls/LayoutTransformControlTestView.xaml

Reply:
I'll take a quick look later and see.  Seems like a bug we should fix for 3.0.  It'll be a separate issue, so we don't necessarily have to hold up this PR over it.

I'll pull the update later and take a look at the new sample.

Reply:
Sample looks good, I created issue #2006 for the bug about the layout.  Think we should port some of the Matrix helper functions and then use those to perform the operation.